packages/devtools/src/schema/types.ts

```diff
@@ -4,10 +4,14 @@ import type {
   ModelRelationsDefinition,
   ModelViewsDefinition,
 } from '@ceramicnetwork/stream-model'
-import type { RuntimeViewField } from '@composedb/types'
+import type { RuntimeViewField, FieldsIndex } from '@composedb/types'
 
 import type { ScalarSchema } from '../types.js'
 
+export type Field = {
```

Comment:
is this redundant of the same-named type from js-ceramic (https://github.com/ceramicnetwork/js-ceramic/blob/8aaec9b820c95d7037eb4a84192ce7eef56072c5/packages/common/src/ceramic-api.ts#L14)?

Can we just import and use the types from js-ceramic instead of duplicating them?

Reply:
exported from js-ceramic now